integ/test_init/test-guide.md

```diff
@@ -0,0 +1,125 @@
+###  Integration Test for Init Process of Fluent Bit on ECS
+
+Following this instruction, you will take the integration test for init process and get the test result
+
+
+
+#### Requirements
+
+1. [install AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
+2. [set up AWS account credential](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-quickstart.html)
+3. [install AWS CDK](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html)
+
+
+
+#### Deploy test resources
+
+* create some Fluent Bit config files locally  (you can use any config files that meet the fluent bit config requirement)
+
+* upload config files to S3 and get ARNs
+
+* create a AWS CDK project
+
+  ```shell
+  mkdir integ_test
+  cd integ_test
+  cdk init app --language python
+  cd integ_test
+  ```
+
+* replace your ARNs in the `integ_test_stack.py` we provided
+
+  **example:**
+
+  ```python
+  # change your arn here
+          arn1="arn:aws:s3:::firelens-example-resources/dummy-input.conf"
+          arn2="arn:aws:s3:::firelens-example-resources/dummy-filter.conf"
+          arn3="arn:aws:s3:::firelens-example-resources/dummy-parser.conf"
+          arn4="arn:aws:s3:::firelens-example-resources/dummy-s3-output.conf"
+          
+  
+  task_definition.add_container("log_router",
+              image=ecs.ContainerImage.from_registry("public.ecr.aws/aws-observability/aws-for-fluent-bit:init-latest"),
+              memory_reservation_mib=50,
+              logging=ecs.LogDrivers.aws_logs(
+                  stream_prefix="Firelens-log",
+              ),
+              environment={ 
+              "aws_fluent_bit_init_s3_1": arn1,
+              "aws_fluent_bit_init_s3_2": arn2,
+              "aws_fluent_bit_init_s3_3": arn3,
+              "aws_fluent_bit_init_s3_4": arn4,
+              },
+          )
+  ```
+
+* copy the `integ_test_stack.py` to override the automatically generated one by AWS CDK (pay attention to the file name)
```

Comment:
copy it where? this step confuses me... it sounds like this python file is autogenerated but then earlier it seems like its the file you are providing...

Reply:
Yea, if you create the CDK project follow this instruction naming it as `integ_test`. It will autogenerated the file`integ_test_stack.py`. 
The easiest way is to delete it and copy the file I provided into that folder instead.

But if you don't name the CDK project as `integ_test`,  for example you name the CDK project as xxxxx, the `xxxxx_stack.py` will be autogenerate, and you need to change the `integ_test_stack.py` I provided to the same name `xxxxx_stack.py` and copy it into the CDK project folder, after that CDK can run normally.

Let me change the sentences in the instruction to explain it more clear